di/container/_utils.py

```diff
@@ -1,22 +1,18 @@
-from typing import Any, List, Mapping
+from typing import List, Mapping, TypeVar
 
-from di.api.dependencies import DependantBase
+T = TypeVar("T")
 
 
-def get_path(
-    dep: DependantBase[Any], parents: Mapping[DependantBase[Any], DependantBase[Any]]
-) -> List[DependantBase[Any]]:
-    path: "List[DependantBase[Any]]" = [dep]
+def get_path(dep: T, parents: Mapping[T, T]) -> List[T]:
+    path: "List[T]" = [dep]
```

Comment:
Just a bit of cleanup to make these functions take generic parameters -> they can be used with plain callable instead of Dependants